src/rust/dbv2/README.md

```diff
@@ -0,0 +1,28 @@
+# Database
+
+This repository provides migrations to create the Econia event database, and Rust types to work with the event database.
+
+The aggregator furthermore adds tables to this database under the `aggregator` schema. Those are disucssed in the `src/rust/aggregator` crate.
```

Comment:
How about a line break after the first sentence?

Also, a typo: `disusssed` -> `discussed`

Reply:
Note that the typo was addressed but not the line break